### PR TITLE
Data Racy Safety Editing

### PR DESCRIPTION
**Pull request by mattmassicotte**
I did a pass over the Data Racy Safety section. It's been a while now and I found many small issues.

I know there's the potential for more changes coming via #50, but this still seems worthwhile in the meantime.

### CODE REVIEW

File: Guide.docc/DataRaceSafety.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ you are doing so statically. Isolation can be a part of these static
 declarations.
 
 There are cases, however, where the type system alone cannot sufficiently
-describe a system's behavior. An example could be an Objective-C type
+describe runtime behavior. An example could be an Objective-C type
```
Comment (mattmassicotte) [inline Guide.docc/DataRaceSafety.md:48]: awkward duplication of the word "system" here with different meaings.

File: Guide.docc/DataRaceSafety.md [hunk 2]
```diff
@@ -130,8 +130,7 @@ access from any other domain.
 
 Actors give the programmer a way to define an isolation domain,
 along with methods that operate within that domain.
-All stored instance properties of an actor are isolated to the enclosing
-actor instance.
+All stored properties of an actor are isolated to the enclosing actor instance.
```
Comment (mattmassicotte) [inline Guide.docc/DataRaceSafety.md:133]: Change wording to "stored properties" to match TSPL and also avoid duplicating the word "instance" within the sentence.

File: Guide.docc/DataRaceSafety.md [hunk 3]
```diff
@@ -226,7 +225,7 @@ but that doesn't mean you must always manually start one.
 Typically, asynchronous functions do not need to be aware of the
 task running them.
 In fact, tasks can often begin at a much higher level,
-within an application framework, or even at the root of a program.
+within an application framework, or even at the entry point of the program.
```
Comment (mattmassicotte) [inline Guide.docc/DataRaceSafety.md:228]: what is "the root of a program"?

File: Guide.docc/DataRaceSafety.md [hunk 4]
```diff
@@ -243,7 +242,7 @@ actor instance, a global actor, or could be non-isolated.
 This isolation can be established manually, but can also be inherited
 automatically based on context.
 Task isolation, just like all other Swift code, determines what mutable state
-they can access.
+is accessible.
```
Comment (mattmassicotte) [inline Guide.docc/DataRaceSafety.md:245]: "they" is quite unclear here.

File: Guide.docc/DataRaceSafety.md [hunk 5]
```diff
@@ -388,7 +387,7 @@ The Swift Programming Language.
 Isolation domains protect their mutable state, but useful programs need more
 than just protection. They have to communicate and coordinate,
 often by passing data back and forth.
-Moving values into or out of an isolation domain is known as crossing an
+Moving values into or out of an isolation domain is known as _crossing_ an
```
Comment (mattmassicotte) [inline Guide.docc/DataRaceSafety.md:390]: New concepts are emphasized throughout this document.

File: Guide.docc/DataRaceSafety.md [hunk 6]
```diff
@@ -406,10 +405,8 @@ They can even be executed in multiple, different domains.
 
 In some cases, all values of a particular type are safe to pass across
 isolation boundaries because thread-safety is a property of the type itself.
-This thread-safe property of types is represented by a conformance to the
-`Sendable` protocol.
-When you see a conformance to `Sendable` in documentation,
-it means the given type is thread safe,
+This is represented by the `Sendable` protocol.
+A conformance to `Sendable` means the given type is thread safe,
```
Comment (mattmassicotte) [inline Guide.docc/DataRaceSafety.md:409]: Lots of redundancy within this sentences. Plus, references "documentation" here is just strange.

File: Guide.docc/DataRaceSafety.md [hunk 7]
```diff
@@ -423,7 +420,7 @@ types in Swift are implicitly `Sendable` when all their stored properties
 are also Sendable.
 However, this implicit conformance is not visible outside of their
 defining module.
-Making a class `Sendable` is part of its public API contract,
+Making a type `Sendable` is part of its public API contract
```
Comment (mattmassicotte) [inline Guide.docc/DataRaceSafety.md:423]: This whole section is about value types!

File: Guide.docc/DataRaceSafety.md [hunk 8]
```diff
@@ -532,16 +529,13 @@ class ChickenValley {
 }
 ```
 
-Being `Sendable`, actor and global-actor-isolated types are always safe
-to pass across isolation boundaries.
```
Comment (mattmassicotte) [inline Guide.docc/DataRaceSafety.md:536]: This is just re-stating information that has appeared immediately before.

File: Guide.docc/DataRaceSafety.md [hunk 9]
```diff
@@ -532,16 +529,13 @@ class ChickenValley {
...
-
 ### Reference Types
 
 Unlike value types, reference types cannot be implicitly `Sendable`.
 And while they can be made `Sendable`,
 doing so comes with a number of constraints.
-To make a class `Sendable`, it must contain no mutable state.
-And any immutable properties must also be `Sendable`.
+To make a class `Sendable` it must contain no mutable state and all
+immutable properties must also be `Sendable`.
```
Comment (mattmassicotte) [inline Guide.docc/DataRaceSafety.md:538]: Awkard wording and structure here.

File: Guide.docc/MigrationGuide.md [hunk 10]
```diff
@@ -31,7 +31,7 @@ Or, you may have been waiting for the Swift 6 release to begin using them.
 Regardless of where your project is in this process, this guide provides
 concepts and practical help to ease the migration.
 
-Here you will find articles and code examples that will:
+You will find articles and code examples here that:
```
Comment (mattmassicotte) [inline Guide.docc/MigrationGuide.md:34]: Really don't like the sound of the duplicated "here"

File: Guide.docc/MigrationGuide.md [hunk 11]
```diff
@@ -43,8 +43,7 @@ Here you will find articles and code examples that will:
 > Important: The Swift 6 language mode is _opt-in_.
 Existing projects will not switch to this mode without configuration changes.
 >
-> There is an important distinction between the _compiler version_
-and _language mode_.
+> There is a distinction between the _compiler version_ and _language mode_.
```
Comment (mattmassicotte) [inline Guide.docc/MigrationGuide.md:46]: This is within a note marked "Important". I don't think restating that this is also important is adding anything.

File: Guide.docc/DataRaceSafety.md (hunk 3 above)
Comment (ktoso) [reply Guide.docc/DataRaceSafety.md:228]: hehe yeah... well, not where the leafs are! 🥬 

Good rewording

**Review by ktoso — APPROVED** [review]

**Pull request merged**